### PR TITLE
added video details page css

### PR DESCRIPTION
**Pull request by Rohanmond**
Preview => https://glex-tv-7tmwnoz37-rohanmond.vercel.app/

### CODE REVIEW

File: src/pages/VIdeoListPage/components/VideoCard/VideoCard.jsx [hunk 1]
```diff
@@ -1,9 +1,15 @@
+import { useNavigate } from 'react-router-dom';
 import './VideoCard.css';
-const VideoCard = ({ element }) => {
-  const { _id, title, creator } = element;
+const VideoCard = ({ video }) => {
+  const navigate = useNavigate();
+  console.log(video, video);
```
Comment (rutvikpumak) [inline src/pages/VIdeoListPage/components/VideoCard/VideoCard.jsx]: remove console

**Review by rutvikpumak — APPROVED** [review]
--clean code
--good variables andwell structured

**Review by romabulani — APPROVED** [review]
- Clean code
- Good folder structure
- Class names are self explanatory
- Good to merge

**Review by anuj-kumary — APPROVED** [review]
- Your class names are self-explanatory and very clean code.
- Good to merge

File: src/pages/VIdeoListPage/components/VideoCard/VideoCard.jsx (hunk 1 above)
Comment (Rohanmond) [reply src/pages/VIdeoListPage/components/VideoCard/VideoCard.jsx]: Thanks

**Pull request merged**